game/effects.py

```diff
@@ -139,9 +139,10 @@ def change_stat(self, target, parameters):
         local_str_fac = parameters.get('strength_factor', 1)
         seq = intervals.Sequence()
         strength = self.strength * local_str_fac
-        strength = max(round(abs(strength) * 0.1), 1)
         if self.strength * local_str_fac < 0:
             strength *= -1
```

Comment:
With out the above `max()` forcing `strength` to be a positive value, we should no longer be doing this manual sign flip.

Reply:
Good idea. So it's probably going to be a while before I get a chance to work on any of this. Would you be able to implement these changes? :)

I agree with your suggested changes. So feel free to make these much needed improvements. That includes the suggestions from back in December too :P